.gitignore

```diff
@@ -25,4 +25,9 @@ test_submissions/submission*
 *.egg-info
 databoard.db
 db_engine.yml
+<<<<<<< HEAD
+prof
+dask-worker-space/
+=======
 coverage.xml
+>>>>>>> upstream/master
```

Comment:
Nice ;)

Reply:
Thanks! Not sure how it got there, didn't see any conflicts..